openhcl/underhill_attestation/src/lib.rs

```diff
@@ -353,9 +368,13 @@ pub async fn initialize_platform_security(
         tcb_version,
     )
     .await
-    .map_err(ErrorInner::GetDerivedKeys)?;
+    .map_err(|e| {
+        tracing::error!("failed to derive keys");
```

Comment:
same question for this

Reply:
This is meant to add an error event to the span, which is separate from the error message.

Reply:
Should this be removed as well?

Reply:
This would create a separate error message, is a little redundant, but it helps with the parsing from the original error message.  I would like to leave this in.  But I do understand that it is redundant to add to the span.